src/index.js

```diff
@@ -21,6 +21,12 @@ export class Animate extends React.Component {
     this.animating = false;
   }
 
+  componentWillMount() {
+    if (this.props.updateOnMount) {
```

Comment:
I'm not sure I follow but I'll talk through my problem. Here's the render method of my class which uses Animate:
```js
return (
  <Animate
    translateX={ this.state.offsetPx }
    opacity={ this.state.opacity }
    style={ width: `${this.state.scalePx}px` }
    tension={ this.props.tension }
    friction={ this.props.friction }
    ref="item"
    updateOnMount
    animate={ this.props.animate }
  >
    { this.props.children }
  </Animate>
);
```

What I'm rendering are pieces of UI on a video player timeline (ticks, labels, current position, etc.). On mount all these items must have an initial `translateX` property.

You're correct that `componentDidMount` calls `triggerAnimations`. But when `triggerAnimations` runs, `this.getChangedProps(lastProps, nextProps)` returns an empty array. Here are my values for `lastProps` and `nextProps`:

```js
lastProps: {
  className: "tick height_0"
}
nextProps: {
  animate: true,
  children: Object,
  friction: 15,
  opacity: 1,
  style: Object,
  tension: 100,
  translateX: -1529.422,
}
```

I'm guessing the `isNumeric` check is keeping my props from showing up as changed.

So anyway, the call to `triggerAnimations` does nothing, and my items have the incorrect `translateX` until the user takes an action that causes them to update.

Are you saying that my default params should be in the `style` hash, rather than asking Animate to set them? I can't remember if I tried that or not.